docs/design_decisions.md

```diff
@@ -220,3 +260,130 @@ Dit betekent dat er in de berichtspecificaties geen gebruik gemaakt wordt van de
 
 ### DD5.9 Properties die gebruik maken van Booleans worden alleen geretourneerd als de waarde 'true' is
 In diverse situaties worden booleans opgenomen als er sprake is van indicatoren. Deze booleans worden alleen geretourneerd als de waarde van de boolean ook informatief is. Dit soort properties worden dus alleen opgenomen als de waarde van de Boolean 'true' is.
+
+### DD5.10 Identificatie van een resource zit altijd op het hoogste niveau van de resource
+De identificatie van een resource zit, wanneer die is opgenomen in de resource anders dan in _links.self, en gebruikt wordt als path-parameter van het resource-endpoint, altijd op het hoogste niveau van de resource in de vorm en inhoud zoals die wordt opgenomen in de uri (path-parameter) van de resource. 
```

Comment:
Het punt is dat je deze identificatie op het hoogste niveau opneemt. Ik weet niet of het punt ook moet zijn dat deze identificatie altijd moet worden opgenomen. Bijvoorbeeld in /ingeschrevenpersonen/{burgerservicenummer}/partners/{id} is in de resource partners geen veld "id" opgenomen. Dit is namelijk puur een technische identificatie, geen functioneel gegeven (zoals bsn en kvknummer dat wel zijn). Misschien moet deze zin hiervoor een beetje herschreven worden, want het kan nu op beide manieren gelezen worden. @JohanBoer mee eens?

Reply:
Is de volgende formulering dan wellicht beter?

> Als de identificatie van de resource geen functioneel gegeven is (zoals burgerservicenummer en kvknummer) maar een technische identificatie en alleen noodzakelijk om gebruikt te kunnen worden als path parameter dan wordt deze opgenomen op het hoogste niveau van de resource. De vorm en inhoud van die identificatie property is dan zoals die wordt opgenomen in de uri (path-parameter) van de resource.

Reply:
Volgens mij was het punt van Frank dat een dergelijk veld soms helemaal niet opgenomen wordt in de resource, maar alleen wordt gebruikt in de link als path parameter. 
Dus **als** er  een identificatie wordt opgenomen dan doe je dat op het hoogste niveau in de resource.

Reply:
Ik heb 'wanneer' vervangen door 'als' en  'anders dan in _links.self,' verwijderd.